### DISCUSSION TITLE
[request] URL name custom (I use Traditional Chinese for Title)

### DISCUSSION THREAD

**Discussion by Peckystudios** [adithya-badidey/tiddlyjam#12] [Ideas]
Hi 

Thank you to open source this great plugin for Tiddlywiki users!

I'm speaking Chinese.

I found when I type a title in Chinese, the blog's URL name would be only a "/"

Is there any way to custom the URL name?

Thanks!

(Exciting the image/media support in the future, can't wait!😄️）

**Comment by ThaddeusJiang** [comment]
@Peckystudios 

I don't user custom URL, but I have made TiddlyJam URL support 中文拼音。
https://thaddeusjiang.github.io/tiddlyjam/tiddlyjam-url-support--zhong-wen-pin-yin.html

repo: https://github.com/ThaddeusJiang/tiddlyjam
